Agoric/ui-kit
# feat: added network notice banner as a resuable component

closes: #114 

## Description



This PR adds `NetworkBanner` (originally implemented in this [PR](https://github.com/Agoric/dapp-psm/pull/94)) as a reusable component for dapps. The purpose of this component is to query the `/network-config` endpoint to find any notices and display them.

The use of this component dapp-econ-gov can be seen in this PR https://github.com/Agoric/dapp-econ-gov/pull/104


**How to test**
- copy `dapp-econ-gov` and `ui-kit` repos and checkout to the `` and `feat/notice-banner` branch respectively
- run `yarn build` in the `ui-kit` repo to create the `dist` directory
- run `yarn add /full/path/to/ui-kit/packages/react-components` in `dapp-econ-gov` to install it as a dependancy.

### Security Considerations



### Scaling Considerations



### Documentation Considerations



### Testing Considerations
